build-tests/rush-redis-cobuild-plugin-integration-test/sandbox/sharded-repo/rush.json

```diff
@@ -0,0 +1,41 @@
+{
```

Comment:
Does anything run this during build, or is this just for local testing?

A README in `build-tests/rush-redis-cobuild-plugin-integration-test` would be helpful.

Reply:
I'd like to hook this up to run during build but I'm not 100% sure how to. Currently, this is for local testing like the `sandbox/repo` folder for cobuilds.